core/baseline-core-v1.0-psd01.md

```diff
@@ -2792,105 +2792,156 @@ As has been done throughout this document, there are BPI layer-specific security
+
+[[R292]](#r292) Testability: Data Orchestration solutions which commit to and preserve operations order utilize a publish/subscribe messaging model where each message is uniquely associated to a topic and where the message identifier is deterministically created. An example of such an implementation with a complete test suite is [Apache Kafka](https://github.com/apache/kafka). Therefore, the requirement is testable.
 
 #### **[R293]**	
 Data Orchestration utilized in a BPI MUST support a consistent state.
 
+[[R293]](#r293) Testability: Data Orchestration solutions that support a consistent state typically utilize a queued publish/subscribe messaging model where each message is uniquely associated to a topic and where the message identifier is deterministically created to preserve state by maintaining a history of state changes. An example of such an implementation with a complete test suite is [Apache Kafka](https://github.com/apache/kafka). Therefore, the requirement is testable.
+
 #### **[R294]**	
-Data Orchestration utilized in a BPI MUST support User-space processing. 
+Data Orchestration utilized in a BPI MUST support user-space processing. 
 
 *In the context of this document, this requirement establishes a capability that allows for one or more computations outside a database to be triggered by a data change in the data storage system.*
 
+[[R294]](#r294) Testability: Data Orchestration solutions that support user-space processing typically utilize a topic-driven publish/subscribe messaging system where messages either trigger outside computations and/or data storage operations or are the result of outside computations and/or data storage operations. An example of such an implementation with a complete test suite is [Apache Kafka](https://github.com/apache/kafka). Therefore, the requirement is testable.
+
 #### **[R295]**	
 Data Orchestration utilized in a BPI MUST NOT make assumptions about consumer uptime.
 
+[[R295]](#r295) Testability: Data Orchestration solutions that make no assumptions about consumer uptime utilize a topic-driven publish/subscribe messaging system where messages are kept in topic queues until they are consumed by a subscribing application (consumer). An example of such an implementation with a complete test suite is [Apache Kafka](https://github.com/apache/kafka). Therefore, the requirement is testable.
+
 #### **[R296]**	
 Data Orchestration utilized in a BPI MUST isolate data source and consumers.
 
+[[R296]](#r296) Testability: Data Orchestration solutions that isolate data source (producer) and consumer utilize a topic-driven publish/subscribe messaging system. An example of such an implementation with a complete test suite is [Apache Kafka](https://github.com/apache/kafka). Therefore, the requirement is testable.
+
 #### **[R297]**	
 Data Orchestration utilized in a BPI MUST support low latency.
 
 *Low latency in this context refers to a latency that does not impact the overall system latency of the BPI.*
```

Comment:
Doesn't this open a potentially circular definition? If the orchestration is part of the BPI, then it supports any latency that doesn't impact its own operation. Would it make sense to clarify that the relevant impact is on the rest of the system?

Reply:
We define overall latency and impact on the system in an earlier section. so it is ok to just focus on this component. @chaals

Reply:
OK. I strongly suggest adding "low latency" and "material impact" to the glossary, and linking to them for requirements like this.

Reply:
ok. will do.